presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -347,11 +348,6 @@ public PlanOptimizers(
                 new LimitPushDown(), // Run the LimitPushDown after flattening set operators to make it easier to do the set flattening
                 new PruneUnreferencedOutputs(),
                 inlineProjections,
-                new IterativeOptimizer(
-                        ruleStats,
-                        statsCalculator,
-                        estimatedExchangesCostCalculator,
-                        new InlineSqlFunctions(metadata, sqlParser).rules()),
```

Comment:
Why do we need to remove this rule?

Reply:
It's not removed. Moved to the beginning of optimizer rules with other desugar expression rules because inline sql functions is conceptually a desugar operation and doing it early would allow more optimizations.